contrib/slime/README.org

```diff
@@ -74,6 +74,15 @@ of Common Lisp, you can specify it in your =~/.spacemacs=
 | ~SPC m c f~ | Compile function         |
 | ~SPC m c r~ | Compile region           |
 
+** Definition
+
+| Key Binding               | Description                     |
+|---------------------------+---------------------------------|
+| ~SPC m d d~               | Go to definition                |
```

Comment:
Sounds good, but SPC m g g is currently used for slime-inspect-definition.
Could move that to SPC m g i.

On Thu, Aug 13, 2015 at 7:27 PM, Sylvain Benner <notifications@github.com>
wrote:

> In contrib/slime/README.org
> <https://github.com/syl20bnr/spacemacs/pull/2648#discussion_r37045468>:
>
> > @@ -74,6 +74,15 @@ of Common Lisp, you can specify it in your =~/.spacemacs=
> >  | ~SPC m c f~ | Compile function         |
> >  | ~SPC m c r~ | Compile region           |
> >
> > +** Definition
> > +
> > +| Key Binding               | Description                     |
> > +|---------------------------+---------------------------------|
> > +| ~SPC m d d~               | Go to definition                |
>
> following CONVENTIONS.md file it should SPC m g g
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/2648/files#r37045468>.
>